[WIP] Add Gang Scheduling Support for lws and volcano implementation example (#498)
kubernetes-sigs/lws: docs/examples/sample/gang-scheduling/README.md

# Enabling Gang Scheduling with Custom Schedulers

> This document provides guidance for how to enable gang scheduling capabilities in LeaderWorkerSet using custom schedulers.

Reviewer: ```suggestion
This document provides guidance about how to enable gang scheduling capabilities in LeaderWorkerSet with different schedulers.
```

# Enabling Gang Scheduling with different Schedulers

This document provides guidance about how to enable gang scheduling capabilities in LeaderWorkerSet with different schedulers.

## Supported Schedulers

| Custom Scheduler Community | GitHub ID       | Email                           |
| -------------------------- |-----------------|---------------------------------|
| Volcano (`@volcano-sh`)    | `@JesseStutler` | `jesseincomparable@hotmail.com` |
| Your-Scheduler-Here        | `@your-id`      | `your@email.com`                |

> We welcome contributions for integrating more custom schedulers that support gang scheduling capabilities. When developing an integration, please:

Author: OK I have modified it, please check
